astroid/__pkginfo__.py

```diff
@@ -2,5 +2,5 @@
 # For details: https://github.com/pylint-dev/astroid/blob/main/LICENSE
 # Copyright (c) https://github.com/pylint-dev/astroid/blob/main/CONTRIBUTORS.txt
 
-__version__ = "3.1.0-dev0"
+__version__ = "3.0.2"
```

Comment:
Don't worry; the tagged 3.0.1 went out with this correctly as 3.0.1. This was a mistake where instead of amending the merge commit to main, I added a commit to the maintenance branch as discussed [here](https://github.com/pylint-dev/astroid/pull/2323#issuecomment-1763576235)

Reply:
Shouldn't this be two commits one with each version then we rebase ? Approving because you probably also thougth of something that work :)

Reply:
The mistake already happened. There shouldn't have been `"3.1.0-dev0"` on this branch at all, so this squash merge will fix a bug. You might be thinking of going from 3.0.2 -> 3.1.0-dev0 on the main branch.